.storybook/preview.tsx

```diff
@@ -3,36 +3,44 @@ import '../src/index.scss'
 import { SeamProvider } from '@seamapi/react'
 import type { Preview } from '@storybook/react'
 
-import { fakePublishableKey, fakeUserIdentifierKey } from './seed-fake.js'
-
-const isProd = process.env['NODE_ENV'] === 'production'
-const useFake = process.env['STORYBOOK_SEAM_ENDPOINT'] == null
+const useFake =
+  process.env['DONT_USE_FAKE'] === '1' &&
+  process.env['STORYBOOK_SEAM_ENDPOINT'] == null
```

Comment:
This won't work since storybook masks all env vars that do not have the storybook prefix. How about?

```suggestion
const useFake =
  ['1','true'].includes(process.env['STORYBOOK_DISABLE_FAKE']?.toLowerCase()) &&
  process.env['STORYBOOK_SEAM_ENDPOINT'] == null
```